### PR TITLE
[Phase 1] Fix&update _race_interstitial modal on dashboard page

### PR DESCRIPTION
**Pull request by levadadenys**
### [Phase 1] Fix&update _race_interstitial modal on dashboard page

Before:
Screenshot from 2023-01-30 15-07-47

After:
Screenshot from 2023-01-30 22-02-12

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by breville** [comment]
Could you add a before screenshot to illustrate the fix?

Also, the screenshot still has a couple possible issues: 
- it seems odd that the body text ("Please tell us...") is smaller than the text for the options.  Normally they'd be the same size?
- it also seems odd that the heading ("Optional: ...") is smaller than the text for the options.

**Comment by levadadenys** [comment]
> Could you add a before screenshot to illustrate the fix?
> 
> Also, the screenshot still has a couple possible issues:
> 
> * it seems odd that the body text ("Please tell us...") is smaller than the text for the options.  Normally they'd be the same size?
> * it also seems odd that the heading ("Optional: ...") is smaller than the text for the options.

Thanks for the hints! Fixed and updated the PR

File: dashboard/app/views/layouts/_race_interstitial.html.haml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #race-modal.modal{'data-backdrop' => 'static', 'data-keyboard' => 'false', style: 'display: none;'}
   .modal-dialog
     .modal-content.no-modal-icon
-      %h2.custom-h2= t('race_interstitial.title')
+      %h2.custom-h2{style: "margin-top: 20px; margin-bottom: 10px; font-size: 1.7em;"}= t('race_interstitial.title')
```
Comment (breville) [inline dashboard/app/views/layouts/_race_interstitial.html.haml]: Do we have a policy on what should be inline styling versus in inline CSS or external SCSS?

**Review by breville — APPROVED** [review]
Styling improvements look good.

File: dashboard/app/views/layouts/_race_interstitial.html.haml (hunk 1 above)
Comment (levadadenys) [reply dashboard/app/views/layouts/_race_interstitial.html.haml]: Good point, thank you. I've moved this styles to interstitial.scss

**Pull request merged**